src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java

```diff
@@ -87,6 +96,9 @@ public void should_return_data_returned_from_the_service() throws Exception {
             .andExpect(jsonPath("$.envelopes[0].zip_deleted").value(envelopes.get(0).zipDeleted))
             .andExpect(jsonPath("$.envelopes[0].rescan_for").value(envelopes.get(0).rescanFor))
             .andExpect(jsonPath("$.envelopes[0].case_number").value(envelopes.get(0).caseNumber))
+            .andExpect(jsonPath("$.envelopes[0].classification")
+            .value(envelopes.get(0).classification.toString().toUpperCase()))
```

Comment:
do you need `toString().toUpperCase()`, can you compare without them?

Reply:
used name() instead now